### DISCUSSION TITLE
How to setup display on SPI1 in raspberry pi pico

### DISCUSSION THREAD

**Discussion by kyrk55** [Bodmer/TFT_eSPI#1242] [Raspberry Pi Pico]
How do i set up lcd for SPI1 i try just change pin to 

#define TFT_MISO  12
#define TFT_MOSI  15
#define TFT_SCLK  14
#define TFT_CS   13
#define TFT_DC   26 
#define TFT_RST  7

but no luck. On SPI0 all work wich no problem.

**Comment by Bodmer** [comment]
This is possible with recent library versions. This is an example for using SPI 1 pins:
```
#define TFT_SPI_PORT 1

#define TFT_MISO  12
#define TFT_MOSI  11
#define TFT_SCLK  10
#define TFT_CS     9  // Chip select control pin
#define TFT_DC     8  // Data Command control pin
#define TFT_RST   15  // Reset pin (could connect to Arduino RESET pin)
```
